src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,24 @@
 package core.basesyntax;
 
+import core.basesyntax.figures.Figure;
+import core.basesyntax.suppliers.FigureSupplier;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
-
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[]{
+                figureSupplier.getRandomFigure(),
+                figureSupplier.getRandomFigure(),
+                figureSupplier.getRandomFigure(),
+                figureSupplier.getDefaultFigure(),
+                figureSupplier.getDefaultFigure(),
+                figureSupplier.getDefaultFigure()
+        };
```

Comment:
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#you-should-create-several-random-figures-so-you-will-use-a-loop-please-dont-create-a-new-figuresupplier-inside-the-loop)
You should create several random Figures, so you will **use a loop**. Please don’t create a new FigureSupplier() inside the loop.
Let’s do it only once - before the loop starts.

Reply:
Fixed